addon/pods/components/frost-tabs/component.js

```diff
@@ -1,24 +1,27 @@
 import Ember from 'ember'
 import layout from './template'
+import PropTypesMixin, { PropTypes } from 'ember-prop-types'
+import uuid from 'ember-simple-uuid'
+
+export default Ember.Component.extend(PropTypesMixin, {
+  // == Component properties ==================================================
 
-export default Ember.Component.extend({
   layout: layout,
   classNames: ['frost-tabs'],
 
-  tabs: [],
+  // == State properties ======================================================
 
-  init () {
-    this._super(...arguments)
-    this.set('tabs', Ember.A([]))
+  propTypes: {
+    tabs: PropTypes.array.isRequired,
+    selectedTab: PropTypes.string,
+    targetOutlet: PropTypes.string,
+    onChange: PropTypes.func,
+    hook: PropTypes.string
```

Comment:
I would like hooks to be made required going forward (so that automation teams have appropriate hooks - we'll get defects otherwise)